doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
         
         .. literalinclude:: code/release.yml     
            :language: yaml
+
+
+.. _organization_secrets:
+
+Organization secrets
+--------------------
+
+  Secrets are encrypted variables that you create in an organization, repository, or repository environment. The secrets that you create are available to use in GitHub Actions workflows. 
+
+  From `GitHub documentation <https://docs.github.com/en/actions/security-guides/encrypted-secrets>`_
+
+
+You can use ``secrets`` to pass sensible data such as passwords, token or IP addresses to your workflows.
+
+By default, ``Ansys`` and ``Ansys-internal`` organizations provide certain ``secrets`` to help you to automate/unify certain tasks such as release.
+
+Actions secrets
+~~~~~~~~~~~~~~~
+
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+| **SECRET**                          | **Repository access**              | **Token?**                            | **Value**                                 | **Description**                                                                              |
```

Comment:
Any reason that SECRET is all caps? Also descripton should likely be lower case to meet the Google style guidelines.

Reply:
No reason indeed.